components/builder-web/app/package/package-builds/_package-builds.component.scss

```diff
@@ -0,0 +1,3 @@
+.hab-package-builds {
+
```

Comment:
Is there a reason this is here and blank?

Reply:
I just wanted to create placeholders for each of these components' styles in a consistent way throughout the module. I suspect these will be used, but as of right now, they aren't strictly necessary.